app/models/merchant.rb

```diff
@@ -23,7 +23,7 @@ def average_item_price
   end
 
   def distinct_cities
-    item_orders.distinct.joins(:order).pluck(:city)
+    (item_orders.distinct.joins(:order).pluck(:city)).sort
```

Comment:
thanks for fixing this one!

Reply:
Might be a better way to do it, maybe on refactor?  But, hopefully no more random failures.